CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+Contributing to k6
+==================
+
+Thank you for your interest in contributing to k6!
+
+(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧
+
+Before you begin, make sure to familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md). If you've previously contributed to other open source project, you may recognize it as the classic [contributor covenant](http://contributor-covenant.org/).
+
+If you want to chat with the team or the community, you can join our Slack team: [LINK GOES HERE].
+
+Filing issues
+-------------
+
+Don't be afraid to file issues! Nobody can fix a bug we don't know exists, or add a feature we didn't think of.
+
+The worst that can happen is that someone closes it right away and points you in the right direction.  
+
+That said, "how do I..."-type questions are often more suited for Slack.
+
+Contributing code
+-----------------
+
+If you'd like to contribute code to k6, this is the basic procedure. Make sure to follow the **style guide** described below!
+
+1. Find an issue you'd like to fix. If there is none already, or you'd like to add a feature, please open one and we can talk about how to do it.
+   
+   Remember, there's more to software development than code; if it's not properly planned, stuff gets messy real fast.
+
+2. Create a fork and open a feature branch based off develop - `feature/my-cool-feature` is the classic way to name these, but it really doesn't matter, as long as you don't hack directly on develop.
```

Comment:
Wait, what? "develop" ? What's that branch and why is that used instead of master? That's highly unorthodox and if you'll keep that system I think you need to explain that here first and not take it for granted. Most git projects out there just use master as... eh, the master branch.